spec/tests/out.txt

```diff
@@ -0,0 +1,8 @@
+......[openstudio.model.StandardGlazing] <0> StandardGlazing::setWindowGlassSpectralDataSetName is deprecated, use StandardGlazing::setWindowGlassSpectralDataSet
```

Comment:
@tanushree04 we should not commit this file to the repo, use git rm to remove it https://git-scm.com/docs/git-rm